src/main/java/domain/Lotto.java

```diff
@@ -34,25 +39,23 @@ private void checkLottoSize(List<LottoNumber> lottoNumbers) {
 		}
 	}
 
-	private void checkDuplicatedLottoNumber(List<LottoNumber> lottoNumbers) {
-		if (lottoNumbers.stream()
-			.distinct()
-			.count() != lottoNumbers.size()) {
+	private void checkDuplicatedLottoNumber(Set<LottoNumber> lottoNumbers) {
+		if (lottoNumbers.size() != LOTTO_SIZE) {
 			throw new IllegalArgumentException("6개의 숫자는 모두 달라야합니다.");
 		}
 	}
 
 	public int calculateMatchCount(Lotto targetLotto) {
 		return (int)lotto.stream()
-			.filter(lottoNumber -> targetLotto.isContain(lottoNumber))
+			.filter(lottoNumber -> targetLotto.contains(lottoNumber))
```

Comment:
메서드 레퍼런스 방식으로 하면 코드가 조금 더 깔끔해져요 :)
```suggestion
			.filter(targetLotto::contains)
```